app/pages/Accounts/useAccountSync.ts

```diff
@@ -71,36 +95,43 @@ export default function useAccountSync(): string | undefined {
             !controller.isAborted
         ) {
             controller.start();
-            dispatch(updateTransactions({ controller, onError: setError }));
+            dispatch(
+                updateTransactions({
+                    controller,
+                    onError: setError,
+                })
+            );
         }
+
+        dispatch(updateTransactions({ controller, onError: setError }));
+
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [
-        account?.address,
-        accountInfo?.accountAmount,
-        account?.status,
-        controller.isAborted,
-    ]);
+    }, [account?.address, accountInfo?.accountAmount, account?.status]);
```

Comment:
I might be misunderstanding what you doing, but the isAborted is in the deps, so the effect is re-run when the previous update has finished.